src/components/Calculator.js

```diff
@@ -1,53 +1,38 @@
 import React from 'react';
 import './Calculator.css';
-import calculate from './logic/calculate';
 
 class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
+  constructor() {
```

Comment:
I have noticed you are still using a class based approach it would be nice if you followed the example below.

// ./src/components/Calculator.js

import React from 'react';

const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);

export default Calculator; // export Calculator component

Kindly refactor your code by using hooks .